tornado/options.py

```diff
@@ -41,6 +41,13 @@ def connect():
     # or
     tornado.options.parse_config_file("/etc/server.conf")
 
+.. note:
+
+   When using tornado.options.parse_config_file, the only options that are set
+   are ones that were previously defined with either tornado.options.parse_command_line
+   or tornado.options.define. tornado.options.parse_config_file is used mainly as a 
+   means to offer overriding settings for different environments.
```

Comment:
You're right (no surprise :)

I think the main point that should be outlined is that you cannot define options with either parse_command_line or parse_config_file. One needs to call define directly and use either, or both, function(s) to override what was set with define.